rpa-home/modules/ROOT/pages/automation-security-securesessions.adoc

```diff
@@ -0,0 +1,50 @@
+= Secure Sessions
+
+In RPA automations, a bot simulates a human user. The bot logs in to a system, sees the screen, and can interact with the UI via key strokes and mouse actions. That way, the bot can even interact with legacy systems, that do not provide an API.
+
+The bot logs in to the computer, where it is installed. But, of course, it does not sit on a chair in front of the desktop. Nevertheless, if you would enter the room where the computer is, you could see what the bot is doing: The screen would show the open applications, the mouse would  move around and typing would appear in forms. In MuleSoft RPA, we call this a _Desktop Session_. To prevent unauthorized data access or manipulation, you would need to guard the physical room.
```

Comment:
```suggestion
The bot logs in to the computer where it is installed. But, of course, it does not sit on a chair in front of the desktop. Nevertheless, if you enter the room where the computer is, you can see what the bot is doing: The screen shows the open applications, the mouse moves around and typing appears in forms. In MuleSoft RPA, we call this a _Desktop Session_. To prevent unauthorized data access or manipulation, you must secure the physical room.
```